python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -225,8 +225,6 @@ class RandomForestClassifier(BaseRandomForestModel,
         Number of bins used by the split algorithm.
         For large problems, particularly those with highly-skewed input data,
         increasing the number of bins may improve accuracy.
-    n_streams : int (default = 4 )
```

Comment:
this unintended deletion was due to improper resolution of merge conflict, will rectify shortly 👍🏾